Source/WebCore/accessibility/AccessibilityObject.cpp

```diff
@@ -3353,8 +3358,11 @@ bool AccessibilityObject::supportsPressed() const
 
 bool AccessibilityObject::supportsExpanded() const
 {
-    // If this object can toggle an HTML popover, it supports the reporting of its expanded state (which is based on the expanded / collapsed state of that popover).
-    if (popoverTargetElement())
+    // Command buttons support expanded, if they can show, hide or toggle a popover.
```

Comment:
Slight rephrasing that might help this read a bit better:

`// Elements with commands should support the expanded state if they can show, hide, or toggle a popover.`

IMO it's OK to not specifically call out buttons in the comment, since there's technically no code local to this function that is button-specific (even though the implementation of `commandForElement` only works for buttons).